spec/fixtures/json/12584148.full.json

```diff
@@ -0,0 +1,191 @@
+  "first_published": null,
+  "created_by": "Admin User",
+  "modified_by": "Admin User",
+  "identifiers": [
+    "cul:jq2bvq851s",
+    "5a25450d-0ed3-48a6-8a96-17a8aaaa3dc6",
+    "clio12584148"
+  ],
+  "title": "Carnegie Corporation Applications No. 1, Carnegie Corporation, 1912",
+  "state": "A",
+  "dc_type": "InteractiveResource",
+  "project": {
+    "pid": "cul:gf1vhhmhbv",
+    "uri": null,
+    "display_label": "Carnegie Digital Past and Future",
+    "short_label": "Carnegie Digital Past and Future",
+    "string_key": "carnegie_dpf"
+  },
+  "publish_targets": [
+    
+  ],
+  "digital_object_type": {
+    "string_key": "item",
+    "display_label": "Item"
+  },
+  "dynamic_field_data": {
+    "collection": [
+      {
+        "collection_term": {
+          "uri": "http:\/\/id.library.columbia.edu\/term\/72fcfd07-f7db-4a18-bdb2-beb0abce071c",
+          "value": "Carnegie Corporation of New York Records",
+          "type": "local",
+          "vocabulary_string_key": "collection",
+          "internal_id": 78,
+          "clio_id": "4079753"
+        }
+      }
+    ],
+    "clio_identifier": [
+      {
+        "clio_identifier_value": "12584148"
+      }
+    ],
+    "marc_005_last_modified": [
+      {
+        "marc_005_last_modified_value": "20190122155950.0"
+      }
+    ],
+    "form": [
+      {
+        "form_term": {
+          "uri": "http:\/\/id.loc.gov\/vocabulary\/graphicMaterials\/tgm001221",
+          "value": "books",
+          "type": "external",
+          "authority": "aat",
+          "vocabulary_string_key": "form",
+          "internal_id": 3
+        }
+      }
+    ],
+    "title": [
+      {
+        "title_sort_portion": "Carnegie Corporation Applications No. 1, Carnegie Corporation, 1912"
+      }
+    ],
+    "type_of_resource": [
+      {
+        "type_of_resource_value": "text"
+      }
+    ],
+    "restriction_on_access": [
+      {
+        "restriction_on_access_value": "Digital version available with no restrictions"
+      }
+    ],
+    "date_created": [
+      {
+        "date_created_start_value": "1912",
+        "date_created_key_date": true
+      }
+    ],
+    "date_created_textual": [
+      {
+        "date_created_textual_value": "1912"
+      }
+    ],
+    "genre": [
+      {
+        "genre_term": {
+          "uri": "temp:2f330fc39fa00bf2bfd3e1fdf511290bffe27ca108e7bb0002067eda7eedc910",
+          "value": "Ledgers (account books)",
+          "type": "temporary",
+          "authority": "aat",
+          "vocabulary_string_key": "genre",
+          "internal_id": 997748
+        }
+      }
+    ],
+    "language": [
+      {
+        "language_term": {
+          "uri": "http:\/\/id.loc.gov\/vocabulary\/iso639-2\/eng",
+          "value": "English",
+          "type": "external",
+          "authority": "iso639-2b",
+          "vocabulary_string_key": "language",
+          "internal_id": 195449
+        }
+      }
+    ],
+    "subject_place": [
```

Comment:
There is no subject_place Hyacinth field.  Maybe you're thinking of subject_geographic, which people sometimes refer to as "subject place", which has been confusing in the past.

Reply:
this is a full hyacinth json object from the web ui

Reply:
Just wanted to note here that I corrected the Hyacinth records that incorrectly contained "subject_place", so this can be removed from the sample doc now.